fineract-provider/src/main/java/org/apache/fineract/interoperation/util/MathUtil.java

```diff
@@ -87,6 +87,7 @@ public static Long abs(Long value) {
 
     /** @return calculates minimum of the two values considering null values
      * @param notNull if true then null parameter is omitted, otherwise returns null */
+    @SuppressWarnings("NullTernary")
```

Comment:
whoa whoa - slow down, the point is to FIX all problems - not to introduce a tool that tells us about problems, and then just make the tool shut up! I have not looked closely, but just a quick glance at https://errorprone.info/bugpattern/NullTernary makes it seem to me like this is trying to telly us about a real bug? As above, if it's definitely a false positive, then put a detailed analysis explanation in an inline comment...

Reply:
Will look more into those remaining two remaining errors and provide if it is necessary to use the @SuppressWarnings.